docker/package/test-ubuntu-binaries.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-FileCopyrightText: 2021 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import subprocess
+import sys
+
+binaries = [
+    "tezos-client",
+    "tezos-admin-client",
+    "tezos-node",
+    "tezos-signer",
+    "tezos-codec",
+    "tezos-baker-PtMumbai",
+    "tezos-accuser-PtMumbai",
+    "tezos-smart-rollup-client-PtMumbai",
+    "tezos-smart-rollup-node-PtMumbai",
+    "tezos-baker-PtNairob",
+    "tezos-accuser-PtNairob",
+    "tezos-smart-rollup-client-PtNairob",
+    "tezos-smart-rollup-node-PtNairob",
+]
+
+
+def setup():
+    commands = [
+        "apt update -y",
+        "apt install -y software-properties-common",
+        "apt update -y",
+        "add-apt-repository -y ppa:serokell/tezos",
+        "apt-get update -y",
+    ]
+    for command in commands:
+        result = subprocess.run(command, shell=True)
+        assert result.returncode == 0, f"{result.returncode}"
+
+
+def test(cleanup: bool):
+    sys.stdout.write("Setting up env\n")
+    setup()
+    sys.stdout.write("Setup is done\n")
+
+    processed_binaries = []
+    try:
+        for binary in binaries:
+            sys.stdout.write(
+                f"---------------------|Testing {binary}|---------------------\n"
+            )
```

Comment:
This output is postponed until the end of the script (you have to manually flush buffer, I suppose). 
BTW, have you considered using something like `pytest`? I'm not insisting at all, it just could be easier in the end.

Reply:
For such simple scripts, I don't think it is needed. For something more complex it might be relevant